oidc-controller/api/authSessions/crud.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, db: Database):
         self._db = db
 
     async def create(self, auth_session: AuthSessionCreate) -> AuthSession:
+        print("auth_session in crud.py create: ", auth_session)
```

Comment:
Might want to remove this, unless we are using a logger. I added a comment to #277 to deal with other print statements as well (we should have non, logger should be used if meaningful).